src/tests/HDFFile.cxx

```diff
@@ -7,10 +7,33 @@
 #include "../helper.h"
```

Comment:
Do you mean the `HDFFile`? Test code has the same name, but in directory `tests`.